test/OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient.Tests/OpenTelemetry.Instrumentation.ElasticsearchClient.Tests.csproj

```diff
@@ -19,7 +19,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\src\OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient\OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient.csproj" />
+    <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient\OpenTelemetry.Instrumentation.ElasticsearchClient.csproj" />
```

Comment:
Shouldn't this have `Contrib` removed as well?

Reply:
I was thinking of renaming the folders in a separate PR to make it a little easier to review.

Reply:
Ok, as far as just changing the namespaces, this looks good to me.